manage.py

```diff
@@ -17,6 +17,10 @@
 CONFIG_FILE.read('app.cfg')
 
 CLIENTS_DATA = yaml.load(open('clients.yaml')) or {}
+# remove deprecated configuration key
+for data in CLIENTS_DATA:
+    if 'redirect_uri' in data:
+        del data['redirect_uri']
```

Comment:
Safer to populate `redirect_uris` if that's not set, but 🤷‍♂️.

Reply:
Yes, because the new configuration cannot be deployed with the existing code; so the new code needs to support both configurations